src/algebra/ordered_field.lean

```diff
@@ -492,11 +492,9 @@ instance linear_ordered_field.to_densely_ordered : densely_ordered α :=
   calc (a₁ + a₂) / 2 < (a₂ + a₂) / 2 : div_lt_div_of_lt two_pos (add_lt_add_right h _)
                  ... = a₂            : add_self_div_two a₂⟩ }
 
-instance linear_ordered_field.to_no_top_order : no_top_order α :=
-{ no_top := λ a, ⟨a + 1, lt_add_of_le_of_pos (le_refl a) zero_lt_one ⟩ }
+instance linear_ordered_field.to_no_top_order : no_top_order α := by apply_instance
```

Comment:
What happens if you completely remove this declaration and the next one?

Reply:
It works!